apps/web/src/components/docs/Docs.tsx

```diff
@@ -105,15 +151,38 @@ export const Docs = ({ code = '', description = '', title = '', isLoading, child
           textAlign: 'justify left',
         })}
       >
-        <div>
-          <Flex justify="space-between" align="center">
-            <TitleH1>{title}</TitleH1>
-            {actions}
-          </Flex>
-          <Text>{description}</Text>
-        </div>
+        <When truthy={!isChildDocs}>
+          <div>
+            <Flex justify="space-between" align="center">
+              <TitleH1
+                className={css({
+                  width: '99%',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Why do we need this?‏

Reply:
Because otherwise the floating close icon and external link will be hiding part of the title

Reply:
Hmm, then there's probably a solution that may address it more directly that would be better to aim for